hack/generate-manifest-windows.sh

```diff
@@ -62,6 +66,10 @@ case $key in
     RUNTIME="containerd"
     shift
     ;;
+    --ovs-run-mode)
```

Comment:
I don't think it is necessary to configure a value for parameter "ovs-run-mode". You can use a paramter like "--include-ovs", the default value for "OVSRunMode" is "service", and it is reset as "pod" when the paramter is set.

Reply:
changed to `--include-ovs`